script/Ciel/Day 10/10_01_CIEL10_5_2.txt

```diff
@@ -52,22 +55,24 @@ Senpai hurries out of the tea ceremony room without waiting for an answer.
 [sha:8d80d3d4e43244569815124a4b7e78f7dd931286]{
 -- Page 385, Offset 28120.
 -- ………………そろそろ１時間経過する。
-Before long, one hour has passed.
+// lint-off:LintStartingEllipsis
+.........It's been about an hour.
```

Comment:
Eh, I feel like this is one of the rare instances where it works to keep the leading ellipsis connected. If the text rendered as it's own string I'd be more for it, but since the reveal will sweep the dots then newline back, I think it's cleaner for it to be part of the same connected 'sentence'